docs/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -358,127 +352,122 @@ export default function ServerComponent() {
-
-If you choose to combine the two approaches, the environment variables will take precedence over values set in the configuration file. One common approach is to set sensitive data (like tokens) in the environment and include everything else in the configuration file added to your VCS.
-
-The URL, organization, and project properties identify your organization and project, and the auth token authenticates your user account.
-
-### Use Configuration Files for Source Map Upload
-
-You can commit all the properties to your VCS, except the auth token. You can accomplish this by using the `.sentryclirc` file and including your auth token:
-
-<OrgAuthTokenNote />
-
-```ini {filename:.sentryclirc}
-[auth]
-token=___ORG_AUTH_TOKEN___
+```javascript {tabTitle:CJS} {filename:next.config.js}
+module.exports = withSentryConfig(nextConfig, {
+  sourcemaps: {
+    disable: true,
+  },
+});
 ```
 
-Don't forget to ignore `.sentryclirc` in your version control system to prevent it from being leaked!
+### Use `hidden-source-map`
 
-### Use Environment Variables
+Depending on your deployment setup, adding `sentry/nextjs` to your app may cause your source code to be visible in browser devtools when it wasn't before. (This happens because of the default behavior of [webpack's `source-map` built-in `devtool`](https://webpack.js.org/configuration/devtool/).) To prevent this, you can use `hidden-source-map` rather than `source-map`, which will prevent your built files from containing a `sourceMappingURL` comment, thus making sourcemaps invisible to the browser. To use `hidden-source-map`, add a `sentry` object to `nextConfig` above, and set the `hideSourceMaps` option to `true`:
 
-Alternatively, the source map upload can be configured using environment variables.
+```javascript {tabTitle:ESM} {filename:next.config.mjs}
+export default withSentryConfig(nextConfig, {
+  hideSourceMaps: true,
+});
+```
 
-| Property    | Environment variable |
-| ----------- | -------------------- |
-| `url`       | `SENTRY_URL`         |
-| `org`       | `SENTRY_ORG`         |
-| `project`   | `SENTRY_PROJECT`     |
-| `authToken` | `SENTRY_AUTH_TOKEN`  |
+```javascript {tabTitle:CJS} {filename:next.config.js}
+module.exports = withSentryConfig(nextConfig, {
+  hideSourceMaps: true,
+});
+```
 
-## Configure Legacy Browser Support
+### Widen the Upload Scope
 
-_(New in version 7.8.0)_
+If you find that there are frames in your client-side stack traces that aren't getting source-mapped even when most others are, it's possible because they are from files in `static/chunks/` rather than `static/chunks/pages/`. By default, such files aren't uploaded because the majority of the files in `static/chunks/` only contain Next.js or third-party code.
```

Comment:
```suggestion
If you find that there are some frames in your client-side stack traces that aren't getting source-mapped even when most others are, the issue might be that those frames are from files in `static/chunks/` rather than `static/chunks/pages/`. By default, such files aren't uploaded because the majority of the files in `static/chunks/` only contain Next.js or third-party code.
```